src/components/GoodsList/GoodsList.jsx

```diff
@@ -0,0 +1,9 @@
+import { GoodsCard } from '../GoodsCard/GoodsCard';
+
+export const GoodsList = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <GoodsCard good={good} />
```

Comment:
where is `key` prop?

Reply:
as the provided array doesn't have the "id" key, I can only use index for the key prop. But in video classes it is sad, not to use them, because they do not work properly.

Reply:
Consider using a good as a key prop. It's unique